src/Evolution/Systems/GeneralizedHarmonic/Characteristics.hpp

```diff
@@ -191,4 +191,25 @@ void compute_evolved_fields_from_characteristic_fields(
     const tnsr::i<DataVector, Dim, Frame>& unit_normal_one_form) noexcept;
 
 // @}
+
+/*!
+ * \ingroup GeneralizedHarmonic
+ * \brief Computes the largest magnitude of the characteristic speeds.
+ *
+ * The function apply() computes the maximum value each characteristic speed
```

Comment:
- I think the brief is sufficient documentation. The order of operations of the implementation is a detail that doesn't need to be part of the public documentation, and the templates (to me) are pretty clear. Please view this comment as an opinion rather than a change request — up to you if you want to change this.

- If you keep the documentation as is, then please fix:
    1. "value each" -> "value of each"
    2. say that you compute the maximum magnitude, rather than just the maximum
    3. note that the tags are templated on dimension and frame, not just dimension.

Reply:
I agree, and I just removed the longer documentation